File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 modul 5/C/PRAK502-2410817110006-HamkaArifani.c
#include <stdio.h>
#include <math.h>
int hitung(int nilai1, int nilai2){
    int Hasil=nilai1-nilai2;
    return (int)fabs((double)Hasil);
}
int mutlak(int angka){
    return (int)fabs((double)angka);
}
int main(){
    int a,b,c,d;
    scanf("%d",&a);
    scanf("%d",&c);
    scanf("%d",&b);
    scanf("%d",&d);
    int Hasil = hitung(a,b) + hitung(c,d);
    printf("%d",mutlak(Hasil));
    return 0;
}

File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 modul 5/Python/PRAk502-2410817110006-HamkaArifani.py
def hitung(angka1,angka2):
    return abs(angka1-angka2)
def mutlak(angka):
    return abs(angka)
a,c,b,d=map(int, input("").split())
hasil=hitung(a,b)+hitung(c,d)
print(mutlak(hasil))